pkg/www/retext/src/components/files/UploadFile.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div>
+        <input type="file" ref="form" v-on:change="upload()">
```

Comment:
It'd be useful to restrict which filetypes can be uploaded (especially for Demographics) via [accept="..."](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)

Reply:
Hmm let me see if I can figure out that api

Reply:
alright tested and it appears to work